app/components/find_interface/phase_banner/view.rb

```diff
@@ -0,0 +1,34 @@
+# frozen_string_literal: true
+
+module FindInterface
+  class PhaseBanner::View < ViewComponent::Base
+    include ViewHelper
+    def initialize(no_border: false)
+      super
+      @no_border = no_border
+    end
+
+    def environment_label
+      Settings.environment.label
+    end
+
+    def environment_colour
+      {
+        "development" => "grey",
+        "qa" => "orange",
+        "review" => "purple",
+        "sandbox" => "purple",
```

Comment:
Can we not share `purple`

Reply:
Discussed offline, decided not to change the colour as the colours are the same accross services.